src/Robo/Blt.php

```diff
@@ -116,6 +120,60 @@ private function addPluginsCommandsAndHooks() {
   }
 
   /**
+   * Adds Symfony (non-Robo) command classes to the application.
+   *
+   * @param \Acquia\Blt\Robo\Application $application
+   */
+  protected function addSymfonyCommands(Application $application) {
+    $twig = new Environment(new FilesystemLoader());
+
+    // Get any custom defined Twig filters to be ignored by linter.
+    $twig_filters = (array) $this->getConfig()->get('validate.twig.filters');
```

Comment:
Just curious, do you see an instance in which we will need to use this array? I see that it is currently empty.

Reply:
If developers add Twig functions or filters to their custom modules and use them in their twig templates, they'll need to add them to their project.yml, otherwise they'll fail validation. I suppose the empty array can be left out of build.yml, and the ability to add functions and filters documented somewhere instead?